Formula/libphonenumber.rb

```diff
@@ -22,16 +22,33 @@ class Libphonenumber < Formula
 
   depends_on "cmake" => :build
   depends_on "googletest" => :build
+  depends_on "abseil"
   depends_on "boost"
   depends_on "icu4c"
   depends_on "protobuf"
   depends_on "re2"
 
+  on_linux do
+    depends_on "gcc"
+  end
+
+  fails_with gcc: "5" # For abseil and C++17
+
+  # Use Homebrew abseil. Patch from Arch, but a similar one is in use by Debian.
+  # A version of this patch is being upstreamed at
```

Comment:
Can we use the version that's being submitted upstream instead of Arch's one?

Reply:
The one that's being upstreamed does not work, but I've left a note there explaining what needs to be changed for it to work.

Reply:
Ok, np.